examples/bri-3/src/bri/communication/communicaton.profile.ts

```diff
@@ -0,0 +1,30 @@
+import { AutomapperProfile, InjectMapper } from '@automapper/nestjs';
+import { createMap, forMember, mapFrom, Mapper } from '@automapper/core';
+import { Injectable } from '@nestjs/common';
+import { BpiMessage } from './models/bpiMessage';
+import { BpiMessageDto } from './api/dtos/response/bpiMessage.dto';
+
+@Injectable()
+export class CommunicationProfile extends AutomapperProfile {
+  constructor(@InjectMapper() mapper: Mapper) {
+    super(mapper);
+  }
+
+  override get profile() {
+    return (mapper) => {
+      createMap(
+        mapper,
+        BpiMessage,
+        BpiMessageDto,
+        forMember(
+          (d) => d.from,
```

Comment:
can we name `from` to FromBpiSubject, since `from` is super unclear? that way we avoid this profile?

Reply:
a bit nitpicky but agree ... unless this leads to changes in a lot of places

Reply:
let's do this in a separate PR. This PR is already huge for review